core/modules/widgets/text.js

```diff
@@ -31,7 +31,9 @@ TextNodeWidget.prototype.render = function(parent,nextSibling) {
 	this.computeAttributes();
 	this.execute();
 	var text = this.getAttribute("text",this.parseTreeNode.text || "");
+	var length = this.getAttribute("length");
```

Comment:
Hello @pmario ,

What do you think about the current implementation?
Seems to be strong enough:
```
parseInt(999,10) || 0;
999
parseInt(0,10) || 0;
0
parseInt(0x1,10) || 0;
1
parseInt(0x10,10) || 0;
16
parseInt(0xff,10) || 0;
255
parseInt("eoeo",10) || 0;
0
parseInt(NAN,10) || 0;
VM1377:1 Uncaught ReferenceError: NAN is not defined(…)(anonymous function) @ VM1377:1
parseInt(NaN,10) || 0;
0
```